### PR TITLE
generalize slugify_ajax

### PR DESCRIPTION
**Pull request by david-venhoff**
### Short description
This pr makes the design of the slug input field of page form, event form and poi form more inline with each other.
Additionally, the methods `slugify_ajax` can now work for pois and events too, while it previously just worked for pages.

### Proposed changes
- Update the slug input field of events
- Make the design of the slug input field uniform across pages, events and pois
- Make the `slugify_ajax` function more generic in order to allow it to also function with events and pois

### Resolved issues
Fixes: #804

### CODE REVIEW

File: src/cms/templates/events/event_form.html [hunk 1]
```diff
@@ -120,20 +120,33 @@ <h1 class="heading">
...
-                        {% spaceless %}
-                            <div style="display: table; white-space: nowrap;">
-                                <span style="display: table-cell;">https://integreat.app/{{ region.slug }}/{{ language.slug }}/events/</span>
-                                {% if event_translation_form.instance.ancestor_path %}
-                                    <span style="display: table-cell;">{{ event_translation_form.instance.ancestor_path }}/</span>
-                                {% endif %}
-                                <span style="display: table-cell; width: 100%;">{% render_field event_translation_form.slug|add_error_class:"slug-error" class="w-full rounded" %}</span>
-                            </div>
-                        {% endspaceless %}
+                    <div id="link-container" class="flex items-center">
```
Comment (melegiul) [inline src/cms/templates/events/event_form.html:127]: Could you please just add a little margin between the two form fields for events and pois?
Depending on the browser there is no/little gap between currently.
On chrome:
Screenshot from 2021-06-21 18-06-54
On firefox:
Screenshot from 2021-06-21 18-07-50

**Review by melegiul — CHANGES_REQUESTED** [review]
Very clean and concise solution, this works as expected :tada:
I could just find a minor problem.

File: src/cms/templates/events/event_form.html (hunk 1 above)
Comment (david-venhoff) [reply src/cms/templates/events/event_form.html]: Sure! I should have tested this with chrome, too.

**Review by melegiul — APPROVED** [review]
Thank you very much
Looks good to me :+1:

**Pull request merged**